cli/server.go

```diff
@@ -1248,3 +1253,21 @@ func startBuiltinPostgres(ctx context.Context, cfg config.Root, logger slog.Logg
 	}
 	return connectionURL, ep.Stop, nil
 }
+
+func handleOauth2ClientCertificates(cfg *codersdk.DeploymentConfig, ctx context.Context) (context.Context, error) {
```

Comment:
In Go, typically `ctx` is the first argument for consistency. The linter pointed this out, but you're also new to Go so figured I'd point it out!

Reply:
Totally makes sense

Change made